Sources/Operation/OriginateAccountOperation.swift

```diff
@@ -5,23 +5,51 @@ import Foundation
 /** An operation that originates a new KT1 account. */
 public class OriginateAccountOperation: AbstractOperation {
   let managerPublicKeyHash: String
+  let contractCode: ContractCode?
 
-  public override var dictionaryRepresentation: [String: String] {
+  public override var dictionaryRepresentation: [String: Any] {
     var operation = super.dictionaryRepresentation
     operation["balance"] = "0"
     operation["managerPubkey"] = managerPublicKeyHash
 
+    if let contractCode = self.contractCode {
+      operation["script"] = [
+        "code": contractCode.code,
+        "storage": contractCode.storage,
+      ]
+    }
+
     return operation
   }
 
-  /** Create a new origination operation that will occur from the given wallet's address. */
-  public convenience init(wallet: Wallet) {
+  /**
+   * Create a new origination operation that will occur from the given wallet's address.
+   *
+   * @param wallet The wallet originating the transaction.
+   * @param contractCode Optional code to associate with the originated contract.
+   */
+  public convenience init(wallet: Wallet, contractCode _: ContractCode? = nil) {
```

Comment:
contractCode isn't used anywhere? why not remove it?

Reply:
This should get passed through.